view/layout.marko

```diff
@@ -42,6 +42,8 @@ html
       div.footer-column
         h3 -- About Us
         p -- Bella Surfaces, Inc. has over 20 years of combined experience in installing custom surfaces in Kitchens, Bathrooms, Playrooms and Commercial facilities throughout the New England area. As seen in "This Old House" Newton, MA Project December 2017. Bella Surfaces, Inc. specializes in countertops, bathroom vanities, bars, fireplaces, reception areas, commercial surfaces, and many, many more. We strive to offer the best customer service in combination with our courteous professional installation teams.
+      `h4 -- Disclaimer
```

Comment:
there's not supposed to be a backtick before the h4

Reply:
Fixed